src/test/resources/null-nested-struct.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" ?>
+<root>
+	<item>
+		<a>a</a>
+		<b>
+			<c>c</c>
+			<d>0.4</d>
+			<es>
+				<e>
+					<ee>1</ee>
+				</e>
+				<e>
+					<ee>2</ee>
+					<eee>3</eee>
+				</e>
+			</es>
+		</b>
+		<f>f</f>
+	</item>
+	<item>
+		<a>a</a>
+		<b>
+			<c>c</c>
+			<d>0.4</d>
+			<es></es>
+		</b>
+		<f>f</f>
+	</item>
+</root>
```

Comment:
I could reproduce the exam exception with the data below:

```xml
<?xml version="1.0" ?>
<root>
  <item>
    <b>
      <es>
        <e>1</e>
      </es>
    </b>
  </item>
  <item>
    <b>
      <es></es>
    </b>
  </item>
</root>
```

(It might be even nicer if the data contains some comments within XML data.)